### ISSUE TITLE
Treat Switch text the same as other free-floating text

### ISSUE THREAD

**Issue by Nancy-Salpepi** [phetsims/circuit-construction-kit-common#799]
**Test device**
MacBook Air (m1 chip)

**Operating System**
12.0.1

**Browser**
Chrome and Safari

**Problem description**
For https://github.com/phetsims/qa/issues/748

I noticed that when a circuit element is behind or very close to the text "Tap circuit element to edit", I can still click on that element. However, when the text for the switch is on the screen, I can't access the element. 

From @arouinfar on slack:
I wonder if the switch text can be treated like other free-floating text. You can click through the “Tap circuit element to edit.” and the “Animation speed limit reached!” text
Perhaps we could do the same for the switch text

**Visuals**
![panelneeded](https://user-images.githubusercontent.com/87318828/144903714-0ac48d66-e97e-4787-9714-b8d60d84f3b6.gif)

**Comment by arouinfar** [comment]
Thanks @Nancy-Salpepi!

@samreid this is a fairly minor issue, so feel free to close as wontfix if my request in https://github.com/phetsims/circuit-construction-kit-common/issues/799#issue-1072484267 isn't feasible.

**Comment by samreid** [comment]
Fixed and ready for cherry-picking.
